app/views/admins/rewards/edit.html.erb

```diff
@@ -2,5 +2,7 @@
 
 <%= render 'form', reward: @reward %>
 
+<%= link_to 'Remove category', remove_category_admins_reward_path(@reward), method: :patch, data: { confirm: 'Are you sure?' } %> |
```

Comment:
as proposed above - maybe simply add here a simple dropdown, and use existing update button instead of creating separate action just for removing category?

Reply:
As above - unnecessary line of code -> removed.